lib/core/network/repository/auth_repository.dart

```diff
@@ -43,6 +43,8 @@ class AuthRepository {
     status.listen((AuthenticationStatus event) {
       if (event is Authenticated) {
         currentAuthStatus = event;
+      } else if (event is UnAuthenticated) {
```

Comment:
we should remove the if completely, Just pass the event to the currentAuthStatus

Both if and else do the same

Reply:
well there's also an unknown status. I guess nobody emits that so sure

Reply:
Actually didn't feel like testing whether unauthorized can come through, but the if could still be better yea

Reply:
I was just assuming there to be a reason for the if statement...